autoload/lsp/ui/vim.vim

```diff
@@ -3,7 +3,7 @@ let s:last_req_id = 0
 function! lsp#ui#vim#definition() abort
     let l:servers = filter(lsp#get_whitelisted_servers(), 'lsp#capabilities#has_definition_provider(v:val)')
     let s:last_req_id = s:last_req_id + 1
-
+    let l:ctx = { 'counter': len(l:servers), 'list':[] }
```

Comment:
I don't think that would address the problem that `jump_if_one` is solving.

`jump_if_one` is a distinction for whether or not we want to go to the location immediately if, after all language servers have answered, we have a singular location.

For `definition` this is true, but for `references` it's false, because I'm assuming one would rarely want to jump blindly with references.